src/events/cd-user-ticket-list-item.vue

```diff
@@ -53,72 +53,31 @@
   </div>
 </template>
 <script>
-  import moment from 'moment';
-  import { sortBy, pick } from 'lodash';
-  import cdDateFormatter from '@/common/filters/cd-date-formatter';
-  import cdTimeFormatter from '@/common/filters/cd-time-formatter';
-  import cdUrlFormatter from '@/common/filters/cd-url-formatter';
+  import { pick } from 'lodash';
   import cdAttendee from './cd-attendee';
+  import EventTile from './cd-event-tile';
   import EventService from './service';
-  import EventsUtil from './util';
 
   export default {
     name: 'user-ticket-list-item',
+    mixins: [EventTile],
     props: ['event', 'dojo', 'usersDojos', 'users'],
     data() {
       return {
         applications: [],
         sessions: {},
         tickets: {},
+        cancelFields: ['dojoId', 'eventId', 'sessionId', 'ticketName',
```

Comment:
Does this really need to be a data prop? Could just have `const cancelFields = [..]` above the export? Just saves the unnecessary tracking of get/set.